docs/self-managed/zeebe-deployment/zeebe-gateway/interceptors.md

```diff
@@ -75,6 +75,64 @@ all incoming calls to the target broker, but it would also be possible to stop
 the message from interception by other interceptors and even to block it from
 dispatch to the broker.
 
+### Implementing a tenant-providing interceptor
+
+:::note
+Camunda 8 relies on Identity for tenant management. Tenant-providing interceptors are only compatible with Zeebe, and
+should only be used when Zeebe is used as a standalone application.
```

Comment:
If Zeebe is used as a standalone application and a user doesn't have access to Helm, do they still need to ensure they have enabled multi-tenancy across all components?

Reply:
No, they won't have to enable multi-tenancy across all components. When Zeebe is used as a standalone application, the assumption is that no other Camunda 8 components are used.

If a user decides to use Zeebe with a custom tenant-providing interceptor and with other C8 components, then multi-tenancy will not work properly, as the other components will rely on Identity for tenant management. As a result, users won't be able to access their data correctly in Operate, Optimize, Tasklist, etc.

Reply:
@christinaausley I documented the above with https://github.com/camunda/camunda-docs/pull/3100/commits/9389f0b14ea83291c24b878556b723bbb0c2f7fa